manifests/controllerconfig.crd.yaml

```diff
@@ -730,13 +760,23 @@ spec:
                                   points to. It is the IP for a self-hosted load balancer
                                   in front of the API servers.
                                 type: string
+                              apiServerInternalIPs:
```

Comment:
We didn't add the list for those platforms in the api[0][1], so I don't think we can. From discussions with other teams, the kubevirt implementation was never completed so we're ignoring that for now. I'm not sure what the status of equinixMetal is, but they don't appear in MCO right now so it seems like that isn't complete either? Or maybe they're using those fields some other way? Either way, if they need support for dual stack vips it will have to be done as a followup because it needs a further api change.

0: https://github.com/openshift/api/blob/2df055c101a3aec29df9589622499ecb69b4437a/config/v1/0000_10_config-operator_01_infrastructure.crd.yaml#L393
1: https://github.com/openshift/api/blob/2df055c101a3aec29df9589622499ecb69b4437a/config/v1/0000_10_config-operator_01_infrastructure.crd.yaml#L432

Reply:
Makes sense. Thanks for the explanation.